sources/packages/web/src/components/common/restriction/AddRestriction.vue

```diff
@@ -51,74 +77,85 @@ export default {
       required: true,
     },
   },
-  setup(props: any, context: any) {
-    const { showDialog, showModal } = useModalDialog<void>();
-    const formData = ref();
-    const formioUtils = useFormioUtils();
-    const formioDataLoader = useFormioDropdownLoader();
+  setup(props: any) {
```

Comment:
Please use the `definedComponent` and remove the `any`.

Reply:
For certain pages, the properties in the component are not created type specific and it takes time to resolve it. So we will create a tech debt to convert the existing one which does not use defineComponent to use it.